eZ/Publish/Core/FieldType/Image/ImageStorage.php

```diff
@@ -122,6 +122,8 @@ public function storeFieldData(VersionInfo $versionInfo, Field $field, array $co
             $field->value->externalData = null;
         }
 
+        if(!isset($field->value->data['id'])){return false;}
```

Comment:
Sorry the if is in the wrong place, it have to be before this line : 
$this->IOService->loadBinaryFile($field->value->data['id']);